scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -184,36 +208,38 @@ def select_feature_set_columns(X, feature_set):
 
 # ## Visualize hyperparameters performance
 
-# In[14]:
+# In[16]:
 
-cv_results_df_dict = {model: 
-    pd.concat([
+def build_cv_results_df(pipeline, feature_set):
+    df = pd.concat([
         pd.DataFrame(pipeline.cv_results_),
-        pd.DataFrame.from_records(pipeline.cv_results_['params']),
-    ], axis='columns') for model, pipeline in cv_pipelines.items()}
+        pd.DataFrame.from_records(pipeline.cv_results_['params'])
+    ], axis='columns')
+    df['feature_set'] = feature_set
+    return df
 
-model = 'full'
+cv_results_df = pd.concat([
```

Comment:
I think a for loop would be more readable than list comprehension here, because you could get rid of the `build_cv_results_df` function. Since it would be inline.